build.gradle

```diff
@@ -829,6 +829,7 @@ project(':core') {
     testImplementation(libs.jfreechart) {
       exclude group: 'junit', module: 'junit'
     }
+    testImplementation libs.mockitoInline // supports mocking static methods, final classes, etc.
```

Comment:
Do we really need this?

Reply:
Yes, I need this to mock a stack method. This library only added in test, not in production, and also used in other tests, I think it should be fine. Any concern about this?

Reply:
Mocking static methods is not ideal, so we should only do it as a last resort.

Reply:
Ok, I'll see if I can test without it. Thanks

Reply:
@ijuma , I've removed the `mockitoInline` library and mock with original Mockito lib. Thanks.